src/adaptors/tcp_adaptor.c

```diff
@@ -34,6 +34,11 @@
 #include <inttypes.h>
 #include <stdio.h>
 
+const uint32_t TCP_MAX_CAPACITY = 131072;
+const uint64_t TCP_SEQ_ACK = 999;  // TBD
```

Comment:
PN_RECEIVED is very similar to this new type, but has two sequences (we could ignore one or the other in the case of TCP should we wish).

One minor upside of using that is that the protocol trace logging in proton already includes the state.